File: app/src/main/java/com/coppermobile/mysampleusingmvvmlivedata/utils/IShowPopUpListener.java
package com.coppermobile.mysampleusingmvvmlivedata.utils;

import android.view.View;

import com.coppermobile.mysampleusingmvvmlivedata.data.Comments;

public interface IShowPopUpListener {

    void showPopupMenu(Comments comments, int position, View view);
}

File: app/src/main/java/com/coppermobile/kotlinsamplemvvmlivedataroom/utils/IShowPopUpListener.kt
package com.coppermobile.kotlinsamplemvvmlivedataroom.utils

import android.view.View

import com.coppermobile.kotlinsamplemvvmlivedataroom.data.Comments

interface IShowPopUpListener {

    fun showPopupMenu(comments: Comments, position: Int, view: View)
}